run-tests.sh

```diff
@@ -416,6 +416,29 @@ elif ! (g3kubectl get pods --no-headers -l app=hatchery | grep hatchery) > /dev/
   donot '@exportToWorkspacePortalHatchery'
 fi
 
+#
+# only run audit-service tests for manifest repos IF audit-service is
+# deployed, and for repos with an audit-service integration.
+#
+runAuditTests=true
+if ! [[ "$service" =~ ^(audit-service|fence|cloud-automation|gen3-qa)$ ]]; then
```

Comment:
isn't this check redundant?

Reply:
No unless i got the logic wrong - if `audit-service|fence|cloud-automation|gen3-qa` then always run the tests, otherwise if `cdis-manifest|gitops-qa|gitops-dev` only run the tests if audit-service is deployed. Don't run the tests for repos that aren't in any of the 2 sets

Reply:
got it, sorry misread it, pls ignore